.github/actions/slack-report/action.yaml

```diff
@@ -0,0 +1,21 @@
+name: "Report Workflow Status to Slack"
+description: "Report the workflow status to Slack to help identify breakages faster."
+inputs:
+  owner:
+    description: "The Slack ID of the person to be notified."
+    default: "U014XCQ9CF8" # @Raymond Kim
```

Comment:
I'm thinking we should send notification to a channel rather than a specific person

Reply:
Yeap it is currently sent into #tt-metal-pipeline-status and Raymond mentioned that we should tag the owner of the test suite (think someone on the dev team that owns Mistral) in addition to sending things to the channel.

This is how it looks in the channel 
<img width="1100" alt="Screenshot 2024-05-13 at 5 02 25 PM" src="https://github.com/tenstorrent/tt-metal/assets/119051828/9e4fc535-7915-4c73-83d4-055165a1f372">

Reply:
Looks great :D

Reply:
Changed the word in the description from `notified` to `tagged`.

Reply:
Does this slack ID refer to the channel or me?

Reply:
This one refers to you. You have indicated that you want to be the default for now. 

I will work on breaking the model tests into different jobs and assigning different owners.

Reply:
I see - I remember but how does it refer to the channel?

Reply:
The part with the channel name is configured in Slack in Workflows - 
<img width="973" alt="Screenshot 2024-05-14 at 12 48 36 PM" src="https://github.com/tenstorrent/tt-metal/assets/119051828/98bfad05-b25e-4303-a8da-9859301508de">

The webhook url that you set up belongs to a particular workflow and the workflow is configured to post to a specific channel.

Reply:
👍